modules/ROOT/pages/setup.adoc

```diff
@@ -113,10 +114,23 @@ The follow the steps explain how to setup the Payload Storage API:
 . Download the pre-packaged API specification (in RAML) and sample implementation template from Anypoint Exchange.
 Install this API on a Mule runtime that can access the storage solution.**[MISSING INFO - link from Anypoint Exchange for the pre-packaged API and integration app from Manik / Pablo]**
```

Comment:
This is not clear for me, I would just remove the the whole "install..." phrase and add the Exchange link in the first one.

Reply:
Fixed.